src/main/java/org/apache/mesos/offer/ExecutorRequirement.java

```diff
@@ -13,6 +13,7 @@
  * have.
  */
 public class ExecutorRequirement {
+    private final Collection<DynamicPortRequirement> dynamicPortRequirements;
     private ExecutorInfo executorInfo;
```

Comment:
May as well mark these final while in here, assuming they likewise aren't being modified anywhere